src/extension/background-script/actions/payments/listByAccount.ts

```diff
@@ -0,0 +1,23 @@
+import type { MessagePaymentListByAccount } from "~/types";
+
+import db from "../../db";
+
+const listByAccount = async (message: MessagePaymentListByAccount) => {
+  const { accountId } = message.args;
+  const limit = message?.args?.limit || 2121;
```

Comment:
Where is 2121 coming from? Is this random or does it have context? Wanna add a comment?

Reply:
it's a nice number. I think I added this because we don't have pagination.

Reply:
:P better add a comment otherwise people might read more into it